packages/jest-config/src/__tests__/normalize.test.ts

```diff
@@ -1614,57 +1628,58 @@ describe('moduleFileExtensions', () => {
   });
 
   it('throws if missing `js` but using jest-runner', () => {
-    [undefined, 'jest-runner'].forEach(runner =>
-      expect(() =>
-        normalize(
-          {
-            moduleFileExtensions: ['json', 'jsx'],
-            rootDir: '/root/',
-            runner,
-          },
-          {},
-        ),
-      ).toThrowError("moduleFileExtensions must include 'js'"),
-    );
-  });
-
-  it('does not throw if missing `js` with a custom runner', () => {
-    expect(() =>
+    [undefined, 'jest-runner'].forEach(
```

Comment:
this needs to `.map` and `await Promise.all`.

probably easier to convert the `it` into `it.each([undefined, 'jest-runner'])`

Reply:
yes I go for convert to `it.each` :)